infra/cifuzz/actions/action.yml

```diff
@@ -5,8 +5,12 @@ inputs:
   project-name:
     description: 'Name of the corresponding OSS-Fuzz project.'
     required: true
+  fuzz-time:
+    description: 'The time total allotted time for fuzzing in seconds.'
+    requried: true
```

Comment:
typo here.

Reply:
Done.

Reply:
The word "time" is still included here unnecessarily.

Reply:
"required" is still misspelled?

Reply:
Done.